ltr/concourse/pipeline.yaml

```diff
@@ -75,6 +75,13 @@ resources:
     bucket: ((readonly_private_bucket_name))
     region_name: eu-west-2
     regexp: search-learn-to-rank/integration-model-(.*).txt
+# AB Test variant B (list_mle_model)
+- name: integration-model-variant-b
+  type: s3-iam
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/integration-model-variant-b-(.*).txt
```

Comment:
I think this regex would clash with the `search-learn-to-rank/integration-model-(.*).txt` one - but since the `(.*)` part is always a timestamp (see https://github.com/alphagov/search-api/blob/master/ltr/concourse/task.sh#L39), these `(.*)`s could be replaced with `([0-9]*)`s